tests/lti/models/auth_data_test.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import json
+import mock
+import pytest
+
+from lti.models.auth_data import AuthData
+
+class TestAuthData(object):
+
+    """Unit tests for the AuthData class."""
+
+    def test_it_loads_json_from_file(self, auth_data):
+        assert auth_data.get_lti_token("93820000000000002") == "9382~IAbeGEFScV  ... IIMaEdK3dXlm2d9cjozd"
+        assert auth_data.get_lti_refresh_token("93820000000000002") == "9382~yRo ... Rlid9UXLhxfvwkWDnj"
+        assert auth_data.get_lti_secret("93820000000000002") == "tJzcNSZadqlHTCW6ow  ... wodX3dfeuIokkLMjrQJqw3Y2"
+        assert auth_data.get_canvas_server("93820000000000002") == "https://hypothesis.instructure.com:1000"
+
+    def test_get_lti_token(self, auth_data):
+        actual = auth_data.get_lti_token("93820000000000002")
+        expected = "9382~IAbeGEFScV  ... IIMaEdK3dXlm2d9cjozd"
+        assert actual == expected
+
+    def test_get_lti_token_throws_key_error_if_oauth_consumer_key_doesnt_exist(self, auth_data):
+        with pytest.raises(KeyError) as ex:
+            actual = auth_data.get_lti_token("KEY_DOES_NOT_EXIST")
+
+    def test_get_lti_refresh_token(self, auth_data):
+        actual = auth_data.get_lti_refresh_token("93820000000000002")
+        expected = "9382~yRo ... Rlid9UXLhxfvwkWDnj"
+        assert actual == expected
+
+    def test_get_lti_refresh_token_throws_key_error_if_oauth_consumer_key_doesnt_exist(self, auth_data):
+        with pytest.raises(KeyError) as ex:
+            actual = auth_data.get_lti_refresh_token("KEY_DOES_NOT_EXIST")
+
+    def test_get_lti_secret(self,auth_data):
```

Comment:
The missing space here `(self,auth_data)` should be causing a lint error, but it's not because linting of the test files isn't working. Will have to fix that later